CONTRIBUTING.md

```diff
@@ -0,0 +1,183 @@
+# Contributing to Puppet modules
+
+So you want to contribute to a Puppet module: Great! Below are some instructions
+to get you started doing that very thing while setting expectations around code
+quality as well as a few tips for making the process as easy as possible. 
+
+## Getting started
+
+- Fork the module repository on GitHub and clone to your workspace
+
+- Make your changes!
+
+## Commit checklist
+
+### The basics
+
+- [x] My commit is a single logical unit of work
+
```

Comment:
I know this is copy-pasted but I'm not sure we need all these newlines - it'd be nice to just have straight lists.